worker/task_test.go

```diff
@@ -53,6 +57,7 @@ var claim = &taskClaim{
 			ClientID:    "abc",
 			Certificate: "",
 		},
+		TakenUntil: tcclient.Time(time.Now().Add(time.Minute * 5)),
```

Comment:
And if you jump through it using something like vimgo, it'll take you to the tcclient package in taskcluster-client-go that shows it's a tcclient.Time, which is really time.Time with a few methods added for serializing/deserializing